x-pack/plugins/cases/public/components/edit_connector/index.test.tsx

```diff
@@ -247,4 +263,39 @@ describe('EditConnector ', () => {
       expect(wrapper.find(`[data-test-subj="push-callouts"]`).exists()).toEqual(true);
     });
   });
+
+  it('disables the save button until changes are done ', async () => {
```

Comment:
Could you please add one more test where you change a field of the same connector?

Reply:
Good idea.

Reply:
After further looking at the code, writing this test is not a simple thing. It requires mocking of many backend APIs and frontend components.

Reply:
Ok! Let's leave it as it is for now.